Network.js

```diff
@@ -1,35 +1,133 @@
-const targetChainId = 97;
-const sourceChainId = 5;
+const { ethers } = require("ethers");
+const fundManagerAbi = require("./artifacts/contracts/upgradeable-Bridge/FundManager.sol/FundManager.json");
+const fiberRouterAbi = require("./artifacts/contracts/upgradeable-Bridge/FiberRouter.sol/FiberRouter.json");
+const tokenAbi = require("./artifacts/contracts/token/Token.sol/Token.json");
+const routerAbi = require("./artifacts/contracts/common/uniswap/IUniswapV2Router02.sol/IUniswapV2Router02.json");
+
+const bscChainId = 97;
+const goerliChainId = 5;
+
+const goerliRPC =
+  "https://nd-018-780-500.p2pify.com/8d55fdf55750fe8f435ef82b610d1bba";
+const bscRPC =
+  "https://nd-409-138-440.p2pify.com/a2b2f87cd496703b1cc64ff8e91b7981";
 
-// FundManagers
 const goerliFundManager = "0x9B887791463cc3BfEBB04D8f54603E5E9ed81f1C"; //proxy
 const bscFundManager = "0xE450A528532FaeF1Feb1094eA2674e7A1fAA3E78"; //proxy
 
-// FiberRouterContract
 const goerliFiberRouter = "0x757FaA8A92b6B813f96058725eC731F75cE0C59f"; //proxy
 const bscFiberRouter = "0x6Cb6Aa70511C9289FbD212E5e320c799Ed2a7Be9"; //proxy
 
-// DEX Routers
 const bscRouter = "0xD99D1c33F9fC3444f8101754aBC46c52416550D1";
 const goerliRouter = "0xEfF92A263d31888d860bD50809A8D171709b7b1c";
 
-// FoundryToken
 const bscUsdt = "0xD069d62C372504d7fc5f3194E3fB989EF943d084";
```

Comment:
Basically, these are just the token addresses that we can use as the refinery/ionic tokens while performing the swaps. 

Currently, USDc is the default foundry. However, cudos & USDT are also foundry assets. 

Regarding, cake or other tokens - since we need to perform test swaps, we can import goerliCake and BScCake as Refinery asset in our swap script call as source/target token.

Reply:
Thanks for explaining @salmanferrum.

Once you make the changes you mentioned in Slack, let me know, and I'll approve and merge this PR.

However, let's connect with @zikriya tomorrow on this item, I have discussed a flow of handling asset types that I think might conflict here. 

@MaiwandSultan if you could assist with organizing this call. Thank you.